.buildkite/data.rayci.yml

```diff
@@ -46,11 +46,11 @@ steps:
       - bazel run //ci/ray_ci:test_in_docker -- //python/ray/data/... //python/ray/air/... data 
         --workers "$${BUILDKITE_PARALLEL_JOB_COUNT}" 
         --worker-id "$${BUILDKITE_PARALLEL_JOB}" --parallelism-per-worker 3
-        --build-name databbuild
-        --test-env RAY_DATA_USE_STREAMING_EXECUTOR=1
+        --build-name data12build
+        --test-env RAY_DATA_USE_STREAMING_EXECUTOR=0
```

Comment:
ah nice catch

Reply:
yeah i think i accidentally flipped this flag in a PR a long time ago, should have been disabled. that's my bad

Reply:
ah i remember why i flipped the flag, otherwise we get a bunch of errors due to timeouts, excessive spilling, and API differences in streaming vs bulk executor. going to confirm with the team if we can deprecate the bulk executor tests

Reply:
we're going to remove the bulk executor tests, yay!

Reply:
w00h00